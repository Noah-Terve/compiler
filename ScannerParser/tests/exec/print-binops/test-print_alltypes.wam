print (5 + 'c');
printc('8' % '9'); /* prints out 8 */
printb(true);
prints("hello world");
printf(5.9 + 5.9);
