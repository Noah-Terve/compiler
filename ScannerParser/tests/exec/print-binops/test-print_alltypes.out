104
8
1
hello world
11.8
